book/src/quickstart.md

```diff
@@ -0,0 +1,83 @@
+# Evaluation Quickstart
+
+This section will guide you through a quick setup of Kanidm for evaluation. It's recommended that
+for a production deployment you follow the steps in the
+[installation chapter](installing_the_server.html) instead as there are a number of security
+considerations you should understand.
+
+### Requirements
+
+- docker or podman
+- `x86_64` cpu supporting `x86_64_v2` OR `aarch64` cpu supporting `neon`
```

Comment:
Didn't we mostly remove the neon bit? 🤔

Reply:
I plan to re-add it in the future, so lets leave it in the docs for now :)